src/snowflake/cli/api/entities/utils.py

```diff
@@ -76,6 +76,7 @@ def _get_stage_paths_to_sync(
     return stage_paths
 
 
+@span("sync_remote_and_local_files")
```

Comment:
this is one of the rare cases where I'd use the actual function name, since it's exactly the logical concept we're timing.